event/Event.proto

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package skywalking.v3;
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.event.v3";
+option csharp_namespace = "SkyWalking.NetworkProtocol.V3";
+option go_package = "skywalking/network/event/v3";
+
+import "common/Common.proto";
+
+service EventService {
+  // When reporting an event, you typically call the collect function twice, one for starting of the event and the other one for ending of the event, with the same UUID.
+  // There are also cases where you have both start time and end time already, for example, when exporting events from a 3rd-party system,
+  // the start time and end time are already known so that you can call the collect function only once.
+  rpc collect (stream Event) returns (Commands) {
+  }
+}
+
+message Event {
+  // Unique ID of the event. Because an event may span a long period of time, the UUID is necessary to associate the
+  // start time with the end time of the same event.
+  string uuid = 1;
+
+  // The object that the event occurs on. In the concepts of SkyWalking, the object is typically service, service instance, etc.
+  string service = 2;
```

Comment:
So, we are going to treat all services as `normal` type, right?

Reply:
Yes I think so

Reply:
I might be nitpicking here but could this be called `source`? The terminology seems a bit closer to what we're looking for here and it would allow more things to be the origin of an event. Something I'm thinking about is if an endpoint is the source of an event. For example, someone could create an event for whenever a certain endpoint hits a certain throughput. Or would that not apply for events and that functionality is implemented somewhere within the alert system?

Reply:
Do you want a structured `Source` with service, instance, and endpoint?

Reply:
@wu-sheng, bingo.

Reply:
@wu-sheng, when you read my comment above did you have any thoughts on the difference between an event and an alert?

Reply:
Alarm could cause an event, which is an abnormal event detected by SkyWalking core. Something similar would be, a VM level metrics alarm event, could relate to the service instance deployed on the VM. 
But in another hand, event doesn't have to be an alarm, like upgrade, HPA, etc.

Reply:
@BFergerson I've added the `Source` object, please take a look